src/components/Dissolution/CompleteResolutionSummary.vue

```diff
@@ -30,7 +30,7 @@
       <router-link
         :to="{ path: `/${RouteNames.DISSOLUTION_RESOLUTION}` }"
       >
-        Return to this step to finish it
+        <soan>Return to this step to finish it</soan>
```

Comment:
Before (note the underlined space before Return):
![image](https://github.com/bcgov/business-create-ui/assets/10002889/99cba6f2-b690-4339-a930-7c09fc6a37d6)

After:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/a96fb204-d8c5-4064-b70c-ff3e605dde2a)